File: Tagx00.Frontend/src/app/models/instance/video/job/VideoWholeJob.ts
import { VideoJob } from "./VideoJob";
import { TagDescriptionTuple } from "../../TagTuple";
import { VideoMissionType } from "../../../mission/video/VideoMission";

export interface VideoWholeJob extends VideoJob {
  type: VideoMissionType.WHOLE;
  tuple: TagDescriptionTuple;
}

File: Tagx00.Backend/src/main/java/trapx00/tagx00/publicdatas/mission/video/whole/VideoWholeJob.java
package trapx00.tagx00.publicdatas.mission.video.whole;

import trapx00.tagx00.publicdatas.mission.TagDescriptionTuple;
import trapx00.tagx00.publicdatas.mission.video.VideoJob;
import trapx00.tagx00.vo.mission.video.VideoMissionType;

public class  VideoWholeJob extends VideoJob {

    private TagDescriptionTuple tuple;

    public VideoWholeJob(VideoMissionType type, TagDescriptionTuple tuple) {
        super(type);
        this.tuple = tuple;
    }

    public TagDescriptionTuple getTuple() {
        return tuple;
    }

    public void setTuple(TagDescriptionTuple tuple) {
        this.tuple = tuple;
    }
}

